### ISSUE TITLE
Firefox support?

### ISSUE THREAD

**Issue by EnricoBDev** [xvrh/puppeteer-dart#333]
Is there any chance we will be able to connect to a geckodriver server instance, i've tried to manually modify the library to connect to it but I was not able to implement this feature

**Comment by caseyryan** [comment]
Agreed, that would be super useful. Some features that don't work in chromium by default work fine in firefox, e.g. video blobs. In chromium I get "media could not be played" but in firefox I get the blob with no problems.
